modules/site/src/app/examples/pagination/pagination-split-usage.component.scss

```diff
@@ -0,0 +1,63 @@
+@import "../../../ng2-material/core/style/variables";
+@import "../../../ng2-material/core/style/default-theme";
+@import "../../../ng2-material/components/whiteframe/whiteframe";
+
+pagination-split-usage {
```

Comment:
Related to this example, can you update it to be viewable on mobile phone size screens? Right now it's too tall and the buttons shift around as you go from page to page.